textinput.go

```diff
@@ -13,11 +13,11 @@ import (
 const textinputChannelName = "flutter/textinput"
 
 // textinputPlugin implements flutter.Plugin and handles method calls to the
-// flutter/platform channel.
+// flutter/textinput channel.
 type textinputPlugin struct {
-	messenger plugin.BinaryMessenger
-	window    *glfw.Window
-	channel   *plugin.MethodChannel
+	messenger   plugin.BinaryMessenger
+	window      *glfw.Window
+	textChannel *plugin.MethodChannel
```

Comment:
I think this was renamed becaues an early version of the PR added a second MethodChannel to this plugin?
Maybe it can be renamed back to simply "channel", to stay consistent with all internal plugins we have.